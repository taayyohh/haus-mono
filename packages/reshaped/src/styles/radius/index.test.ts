import getRadiusStyles from "./index";

describe("Styles/Radius", () => {
	test("handles value", () => {
		expect(getRadiusStyles("large")).toMatchSnapshot();
	});

	test("handles none", () => {
		expect(getRadiusStyles("none")).toMatchSnapshot();
	});

	test("handles undefined value", () => {
		expect(getRadiusStyles()).toMatchSnapshot();
	});

	test("handles responsive value", async () => {
		expect(getRadiusStyles({ s: "small", m: "none", l: "circular" })).toMatchSnapshot();
	});
});
