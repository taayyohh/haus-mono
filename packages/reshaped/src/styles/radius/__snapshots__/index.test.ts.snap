// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Styles/Radius > handles none 1`] = `
{
  "classNames": [
    "_root_979529",
    "_--radius-none_979529",
  ],
}
`;

exports[`Styles/Radius > handles responsive value 1`] = `
{
  "classNames": [
    "_root_979529",
    "_--radius-small_979529",
    "_--radius-none--m_979529",
    "_--radius-circular--l_979529",
  ],
}
`;

exports[`Styles/Radius > handles undefined value 1`] = `null`;

exports[`Styles/Radius > handles value 1`] = `
{
  "classNames": [
    "_root_979529",
    "_--radius-large_979529",
  ],
}
`;
